File: Data Structures/Arrays/Cpp/Calculate_the_sum.cpp
// Write a program to read an array and find the sum of the elements.

#include<iostream>
using namespace std;

int main(){
    int Array[10];
    for(int i = 0; i < 10; i++){
        cin >> Array[i];
    }
    int sum = 0;
    for(int i = 0; i < 10; i++){
        sum += Array[i];
    }
    cout << "The Sum of the elements of the Array: " << sum << endl;
    return 0;
}

File: Data Structures/Arrays/Java/Calculate_the_sum.java
// Write a program to read an array and find the sum of the elements.

import java.util.Scanner;

public class Calculate_the_sum{
    public static void main(String[] args){
        Scanner read = new Scanner(System.in);
        int[] array = new int[10];
        for(int i = 0; i < 10; i++){
            array[i] = read.nextInt();
        }
        int sum = 0;
        for(int i = 0; i < 10; i++){
            sum += array[i];
        }
        System.out.println("The sum of the elements of the array is: " + sum);
        read.close();
    }
}